### PR TITLE
New Term: C++ String `.replace()`

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From Target Keyword list

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/strings/terms/replace/replace.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.replace()` function returns a string with a portion replaced by another string, or a portion of another string.
+
+## Syntax
+
+```pseudo
+basestring.replace(pos, len, str, subpos, sublen)
```
Comment (KTom101) [inline content/cpp/concepts/strings/terms/replace/replace.md:20]: Can we use camelcase for "basestring" in the syntax? I feel it's a little bit easier to read.
```suggestion
baseString.replace(pos, len, str, subpos, sublen)
```

File: content/cpp/concepts/strings/terms/replace/replace.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+- `basestring` is the string whose contents are being replaced.
```
Comment (KTom101) [inline content/cpp/concepts/strings/terms/replace/replace.md:23]:
```suggestion
- `baseString` is the string whose contents are being replaced.
```

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski I only see one thing I would change.

**Comment by SSwiniarski** [comment]
@KTom101, I went and committed your suggestion.

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Let's get this to the 2nd review!

File: content/cpp/concepts/strings/terms/replace/replace.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```pseudo
+baseString.replace(pos, len, str, subpos, sublen)
+```
+
+- `baseString` is the string whose contents are being replaced.
+- `pos` is the zero-based starting position of the replacement.
+- `len` is the number of characters to be replaced.
+- `str` is the replacement string.
+- `subpos` is optional, and is the zero-based starting point of the replacement characters used from `str`.
+- `sublen` is optional, and is the number of replacement characters used from `str`
```
Comment (Dusch4593) [inline content/cpp/concepts/strings/terms/replace/replace.md:28]:
```suggestion
- `sublen` is optional, and is the number of replacement characters used from `str`.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Just had one suggestion but nothing else to add 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, done!

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! 👍🏻

**Pull request merged**